src/main/webapp/WEB-INF/mvc-dispatcher-servlet.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<beans xmlns="http://www.springframework.org/schema/beans"
+       xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xmlns:context="http://www.springframework.org/schema/context"
+       xsi:schemaLocation="http://www.springframework.org/schema/beans
+        http://www.springframework.org/schema/beans/spring-beans-4.3.xsd
+        http://www.springframework.org/schema/context
+        http://www.springframework.org/schema/context/spring-context-4.3.xsd">
+
+    <context:spring-configured/>
+    <context:annotation-config />
+    <context:load-time-weaver/>
+
+    <bean id="mainController" class="com.epam.jmp2017.controller.MainController"/>
+
+    <alias name="conditionsLoader" alias="loader" />
+    <bean id="conditionsLoader" class="com.epam.jmp2017.util.loaders.ConditionsLoader" init-method="cacheConditions">
```

Comment:
Explicit declaration will make this bean recognizable. At least you will think a bit before changing scope for this bean (for ex. during refactoring).